### ISSUE TITLE
Update HTML components to MUI: ./src/components/manageProjects/eventForm.js

### ISSUE THREAD

**Issue by JackHaeg** [hackforla/VRMS#1699]
### Overview
Review all components in`./src/components/manageProjects/eventForm.js` and replace all standard HTML components with applicable MUI components.

### Action Items
- [ ] Review components in:`./src/components/manageProjects/eventForm.js` and search for any standard HTML components.
  - _IF all components are already updated to relevant MUI_
    - [ ] Comment on this issue that all components are already updated to MUI. 
    - [ ] Close the issue & check the box associated with this issue on the [MUI issue epic](https://github.com/hackforla/VRMS/issues/1617)
    - [ ] Move on to the next [MUI issue](https://github.com/hackforla/VRMS/issues/1617) 
  - _IF standard HTML components are found in the file:_ 
    - [ ] Replace these standard HTML components with applicable MUI components. 
    - [ ] Make a PR for this file

### Resources
- https://mui.com/material-ui/
- If you run into any questions, add a comment to the issue and tag "@trillium"

**Comment by sasszz** [comment]
Event form is located in a few places including Adding/Editing events in a Project. Example screenshot below:

<img width="389" alt="Image" src="https://github.com/user-attachments/assets/0053d249-e094-4445-a97b-df4ccb20b27c" />

Components to be converted should be Modal, Input, Select and Button

**Comment by sasszz** [comment]
@JackHaeg @trillium Hi, question, the MUI linked is for "Base UI" which is now deprecated. A different PR used Material UI ( mui #1813 that fixes https://github.com/hackforla/VRMS/issues/1694), which package for MUI should I use?

<img width="1048" alt="Image" src="https://github.com/user-attachments/assets/199c7edd-12c2-4e9d-aaea-98788a31f043" />

**Comment by sasszz** [comment]
<img width="650" alt="Image" src="https://github.com/user-attachments/assets/b5de5147-c99b-48f0-849d-212999f962dd" />

**Comment by trillium** [comment]
Hey, please use the non-depricated package. If we need to install he correct MUI dependencies, please update package.json and yarn.lock in their own commit.

**Comment by sasszz** [comment]
Hey thanks for the reply! 

@trillium should we plan to use:
- Material MUI
    - used in the last pr that was merged
- Base UI
    - Most similar to the deprecated package of MUI Base which is the one linked in the ticket and other related issues
    - introduces an additional component package to the repo that may be redundant

**Comment by trillium** [comment]
Hey @sasszz, happy to clarify :)

Use Material MUI, the Base UI was a mistakely added resource

**Comment by JackHaeg** [comment]
**Hi @sasszz! When you have a moment, can you please provide an update with the following information:**
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures or links* (if necessary): "Add any pictures or links that will help illustrate what you are working on."
  - remember to add links to the top of the issue if they are going to be needed again.

**Comment by sasszz** [comment]
Hey Jack!

Draft PR open here: https://github.com/hackforla/VRMS/pull/1844

1. Progress - 50% - scoped out issue, understand component, about 1/2 through refactoring. Although it's one form, it has multiple components (Modal, Input, Select and Button), making it a larger ticket (more like 0.5)
2. Difficulties - I have 3:
    - Global CSS applied to all inputs make transition to MUI components tricky as the refactored MUI component has extra CSS acting upon it, resulting in weird dimensions/padding.
    - Should we consider extracting for example, the `<Input>` component to something reusable in other forms so we don't have to recreate it?
    - How close should I follow the styles of the prev component (pre-refactor)
3. Availability - 10 hours this week
4. If I could get advice on these blockers, I could get a final PR up EOW

**Comment by sasszz** [comment]
@trillium just to confirm, adhering to current styles is not necessary? Maybe aim for close to the original component styles, but not exact?

**Comment by trillium** [comment]
Correct, for this component feel free to use your judgement on what needs to change

**Comment by sasszz** [comment]
Update!

    Progress: PR is completed and ready for review #1844 
    Blockers: Awaiting other dev approvals
    Availability: Ready for new task
    ETA: This week, after review process is completed

**Comment by sasszz** [comment]
Update!
```
Progress: PR has one approval and is awaiting 1 more
Blockers: Awaiting other dev approval
Availability: Ready for new task
ETA: Today after today's meeting
```
